includes/sanitizers/class-amp-srcset-sanitizer.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * Class AMP_Srcset_Sanitizer.
+ *
+ * @package AMP
+ */
+
+/**
+ * Class AMP_Srcset_Sanitizer
+ *
+ * Sanitizes the `srcset` attribute of elements.
```

Comment:
For the docs, the short description is shown in the indexes. We should get into the habit of making the short description useful, instead of just repeating the name of the element, which is redundant.
```suggestion
 * Sanitizes the `srcset` attribute of elements.
```